srcpkgs/python3-pep517/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'python3-pep517'
+pkgname=python3-pep517
+version=0.9.1
+revision=1
+wrksrc=pep517-${version}
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+depends="python3 python3-toml"
+# tests need flit_core v2, but we have v3 packaged, remove pip when this is fixed
+checkdepends="python3-pytest python3-testpath python3-mock python3-pip"
```

Comment:
If we lack a mandatory dependency tests should be disabled. You can (and should) leave a comment about why, but we don't want to download packages with pip in any situation.